adbms/src/test/scala/de/up/hpi/informationsystems/adbms/csv/CSVParserTest.scala

```diff
@@ -13,15 +14,15 @@ import org.scalatest.{Matchers, WordSpec}
 object CSVParserTest {
   
   object TestRelation extends RelationDef {
-    val byteCol: ColumnDef[Byte] = ColumnDef("byteCol")
-    val shortCol: ColumnDef[Short] = ColumnDef("shortCol")
-    val intCol: ColumnDef[Int] = ColumnDef("intCol")
-    val longCol: ColumnDef[Long] = ColumnDef("longCol")
-    val floatCol: ColumnDef[Float] = ColumnDef("floatCol")
-    val doubleCol: ColumnDef[Double] = ColumnDef("doubleCol")
-    val charCol: ColumnDef[Char] = ColumnDef("charCol")
-    val stringCol: ColumnDef[String] = ColumnDef("stringCol")
-    val boolCol: ColumnDef[Boolean] = ColumnDef("boolCol")
+    val byteCol: ColumnDef[Byte] = ColumnDef[Byte]("byteCol")
```

Comment:
I **don't** hate having to specifically tell the factory which type `Column` I want to define, so I wouldn't put any work into trying to figure out why `scala` needs this rn.

Maybe we can get rid of the type annotations for the values though, right? It says the type right there in the call to the factory...

Reply:
This is due to Intellij complaining about:
> public members should have type annotations

It's not needed by the compiler, right.

Reply:
Oh alright, then we should just keep that. It's nice if the code doesn't look like a christmas tree with standard linting.